<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRCodeGenerationGUI</class>
 <widget class="QWidget" name="QRCodeGenerationGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QR Code Generation</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_main">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_left">
     <item>
      <widget class="QGroupBox" name="categoryGroupBox">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>600</height>
        </size>
       </property>
       <property name="title">
        <string>QR Code Type</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_category">
        <item>
         <widget class="QComboBox" name="categoryComboBox">
          <item>
           <property name="text">
            <string>Text</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>URL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Email</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Phone Number</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SMS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>WiFi</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Contact</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Calendar</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Geo Location</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="parametersGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>200</height>
        </size>
       </property>
       <property name="title">
        <string>Parameters</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_params">
        <item>
         <widget class="QStackedWidget" name="parameterStackedWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="emptyPage"/>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonLayout">
       <item>
        <widget class="QPushButton" name="generateButton">
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_right">
     <item>
      <widget class="QGroupBox" name="outputGroupBox">
       <property name="title">
        <string>QR Code Output</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_output">
        <item>
         <widget class="QLabel" name="qrCodeLabel">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>350</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>QR Code will appear here</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="outputButtonLayout">
          <item>
           <widget class="QPushButton" name="copyButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Copy Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Save Image</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="contentPreviewEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>QR code content preview</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
